odoo/addons/base/models/res_company.py

```diff
@@ -198,37 +198,41 @@ def _company_default_get(self, object=False, field=False):
     def cache_restart(self):
         self.clear_caches()
 
-    @api.model
-    def create(self, vals):
-        if not vals.get('favicon'):
-            vals['favicon'] = self._get_default_favicon()
-        if not vals.get('name') or vals.get('partner_id'):
+    @api.model_create_multi
+    def create(self, vals_list):
```

Comment:
I have to say that this one is not clearer after the "batch", which is not a batch as we create each item from the list one by one ... Can't we keep the old implementation or find a way to do it nicely in batch ?